dev-python/meson-python/meson_python-0.13.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Short description of ProjectX"
+DESCRIPTION="``meson-python`` is a Python build backend built on top of the Meson__
+build system. It enables to use Meson for the configuration and build
+steps of Python packages. Meson is an open source build system meant
+to be both extremely fast, and, even more importantly, as user
+friendly as possible. ``meson-python`` is best suited for building
+Python packages containing extension modules implemented in languages
+such as C, C++, Cython, Fortran, Pythran, or Rust. Consult the
+documentation__ for more details."
```

Comment:
Is Markdown formatting supported in the description? Otherwise, this should be cleaned up.